interfaces/acados_matlab/acados_ocp.m

```diff
@@ -12,42 +12,42 @@
 	methods
 
 
-		function obj = acados_ocp(model, opts)
+		function obj = acados_ocp(model, opts, build_dir)
 			obj.model_struct = model.model_struct;
 			obj.opts_struct = opts.opts_struct;
+      
+      if nargin <= 2
+        build_dir = fullfile(pwd, 'build');
```

Comment:
default values for the options should be set in the `acados_ocp_opts` class

Reply:
resolved